composer.json

```diff
@@ -42,7 +42,7 @@
         "phpstan/phpstan-phpunit": "^1.0",
         "phpunit/phpunit": "^9.5",
         "psalm/plugin-phpunit": "^0.17",
-        "psalm/plugin-symfony": "^3.1",
+        "psalm/plugin-symfony": "^4.0.1",
```

Comment:
Why not using the version 5?

Reply:
The goal of this PR is to fix the CI and this is the smallest upgrade which does it.

Bumping psalm/plugin-symfony would require to bump vimeo/psalm as well and would potentially lead to other things to do so  I think it would be better in another PR.

Reply:
I hope you will do it then! :wink: